pages/tutorials/3.0/graphics/shape.md

```diff
@@ -91,7 +102,12 @@ rectangle.setSize(sf::Vector2f(100.f, 100.f));
 
```

Comment:
Example code above could be adjusted to use `{...}` instead of `sf::Vector2f(...)`

Reply:
Good, that's my personal preference as well. I'll make those same changes in future PRs.